uniffi_bindgen/src/interface/function.rs

```diff
@@ -81,7 +81,11 @@ impl Function {
         &self.ffi_func
     }
 
-    pub fn throws(&self) -> Option<&str> {
+    pub fn throws(&self) -> bool {
```

Comment:
is this a breaking change? since `Function` (and I guess `Method` and `Constructor`) are exposed publically from `uniffi_bindgen::interface` and we are changing  the signature of the `throws` function - should we have changelog entry to represent that?

Reply:
Good point, I just updated the changelog.